doc/source/contributing_code.rst

```diff
@@ -213,18 +216,48 @@ and activated your virtual environment. First we need to install dependencies. I
             matplotlib mmtf-python mock netcdf4 networkx \
             "numpy>=1.17.3" psutil pytest scikit-learn scipy \
             "seaborn>=0.7.0,<0.9" sphinx==1.8.5 "tidynamics>=1.0.0" \
```

Comment:
Not sure where the <0.9 for seaborn came from, maybe py2.7? Anyways, CI doesn't have a max limit.
We got rid of the sphinx limit, so we don't have to limit it to 1.8.5 anymore.

```suggestion
            "seaborn>=0.7.0" sphinx "tidynamics>=1.0.0" \
```